config/core/resources/pingsource.yaml

```diff
@@ -170,6 +170,9 @@ spec:
     - name: Sink
       type: string
       jsonPath: .status.sinkUri
+    - name: EventsSchedule
```

Comment:
`Events` is redundant. Just use `Schedule`

Reply:
done !!!